packages/manifest/src/StatsManager.ts

```diff
@@ -256,7 +256,10 @@ class StatsManager {
       const { name, manifest: manifestOptions = {} } = this._options;
 
       const metaData = this._getMetaData(compiler, compilation, extraOptions);
-
+      if (this._options.getPublicPath) {
```

Comment:
It can be moved here https://github.com/module-federation/core/pull/2638/files#diff-d6f6a50c1270fc71c19737e8785ec98fba4273385a2f3c2213e061683b4c9b48L131

publicpath and getPublicPath should only one of these two can exist

Reply:
Okay done